vyper/parser/parser.py

```diff
@@ -569,32 +707,52 @@ def parse_func(code, sigs, origcode, global_ctx, _vars=None):
+                            default_copiers.append(get_arg_copier(
+                                sig=sig,
+                                memory_dest=var.pos,
+                                total_size=_size,
+                                offset=_offset,
+                            ))
                         else:
                             # Add clampers.
-                            default_copiers.append(make_clamper(calldata_offset - 4, var.pos, var.typ))
+                            default_copiers.append(make_clamper(
+                                calldata_offset - 4,
+                                var.pos,
+                                var.typ,
+                            ))
                             # Add copying code.
                             if isinstance(var.typ, ByteArrayLike):
                                 _offset = ['add', 4, ['calldataload', calldata_offset]]
                             else:
                                 _offset = calldata_offset
-                            default_copiers.append(get_arg_copier(sig=sig, memory_dest=var.pos, total_size=var.size * 32, offset=_offset))
+                            default_copiers.append(get_arg_copier(
+                                sig=sig,
+                                memory_dest=var.pos,
+                                total_size=var.size * 32,
+                                offset=_offset,
+                            ))
 
                     # Unpack byte array if necessary.
                     if dynamics:
                         i_placeholder = context.new_placeholder(typ=BaseType('uint256'))
                         for idx, var_pos in enumerate(dynamics):
                             ident = 'unpack_default_sig_dyn_%d_arg%d' % (default_sig.method_id, idx)
-                            default_copiers.append(
-                                make_unpacker(ident=ident, i_placeholder=i_placeholder, begin_pos=var_pos)
-                            )
+                            default_copiers.append(make_unpacker(
+                                ident=ident,
+                                i_placeholder=i_placeholder,
+                                begin_pos=var_pos,
+                            ))
                     default_copiers.append(0)  # for over arching seq, POP
 
                 sig_chain.append([
                     'if', sig_compare,
                     ['seq',
                         private_label,
-                        LLLnode.from_list(['mstore', context.callback_ptr, 'pass'], annotation='pop callback pointer', pos=getpos(code)) if sig.private else ['pass'],
+                        ['pass'] if not sig.private else LLLnode.from_list([
+                            'mstore',
+                            context.callback_ptr,
+                            'pass',
+                        ], annotation='pop callback pointer', pos=getpos(code)),
```

Comment:
Explain this change please

Reply:
My guess is that it made the linter happier *or* it seems a bit clearer to indicate that the expression is ternary by putting the syntax for that on the first line.

Reply:
Read it again with a clearer head, just switched the order of the if..then..else statement